dlt/pipeline/pipeline.py

```diff
@@ -1114,38 +1117,29 @@ def _optional_sql_job_client(self, schema_name: str) -> Optional[SqlJobClientBas
             logger.info("Client not available due to missing credentials")
         return None
 
-    def _restore_state_from_destination(self, raise_on_connection_error: bool = True) -> Optional[TPipelineState]:
+    def _restore_state_from_destination(self) -> Optional[TPipelineState]:
         # if state is not present locally, take the state from the destination
         dataset_name = self.dataset_name
         use_single_dataset = self.config.use_single_dataset
         try:
             # force the main dataset to be used
             self.config.use_single_dataset = True
-            job_client = self._optional_sql_job_client(normalize_schema_name(self.pipeline_name))
-            if job_client:
-                # handle open connection exception silently
-                state = None
-                try:
-                    job_client.sql_client.open_connection()
-                except Exception:
-                    # pass on connection errors
-                    if raise_on_connection_error:
-                        raise
-                    pass
-                else:
-                    # try too get state from destination
-                    state = load_state_from_destination(self.pipeline_name, job_client.sql_client)
-                finally:
-                    job_client.sql_client.close_connection()
-
-                if state is None:
-                    logger.info(f"The state was not found in the destination {self.destination.__name__}:{dataset_name}")
+            schema_name = normalize_schema_name(self.pipeline_name)
+            with self._maybe_destination_capabilities():
+                schema = Schema(schema_name)
+                schema.update_normalizers()
```

Comment:
it is not a bad idea to do that but IMO all should work without it.  the schema should be created with naming convention coming from current destination caps. also see this method `_set_context` in Piepline. it is called in dlt.pipeline() when new instance is activated

Reply:
this somehow does not work properly. If you create a pipeline and then sync with destination, the resulting schema does not have the right normalisers set.